chia/wallet/util/debug_spend_bundle.py

```diff
@@ -12,7 +12,7 @@
 from chia.util.hash import std_hash
 from chia.wallet.uncurried_puzzle import UncurriedPuzzle
 
-CONDITIONS = {k: bytes(v)[0] for k, v in ConditionOpcode.__members__.items()}  # pylint: disable=E1101
+CONDITIONS = {k: bytes(v)[0] for k, v in ConditionOpcode.__members__.items()}
```

Comment:
i think this stuff that was being complained about was also not needed.  but, i'll explore that in another pr.  just fyi that a real smell was being pointed out here.

https://github.com/Chia-Network/chia-blockchain/pull/18767